pkg/eventcollector/triggers.go

```diff
@@ -34,7 +35,7 @@ func (n *AddNewsroomWatchersTrigger) Run(collector *EventCollector,
 	if !n.ShouldRun(collector, event) {
 		return errors.New("AddNewsroomWatchersTrigger should not run")
 	}
-	newsroomAddr := event.LogPayload().Address
```

Comment:
I was looking through events and it seems that this address doesn't correspond to the "ListingAddress" so it didn't get any new events

Reply:
Thanks good catch on this.